src/recordlinker/database/mpi_service.py

```diff
@@ -367,3 +368,11 @@ def delete_persons(
         session.delete(person)
     if commit:
         session.commit()
+
+
+def check_person_for_patients(session: orm.Session, person: models.Person) -> int | None:
```

Comment:
Given the docstring, what do you think about this change?
```suggestion
def check_person_for_patients(session: orm.Session, person: models.Person) ->bool:
```

Reply:
I think that was my original intention, because you're right, it is more aligned with the docstring.